algebra/BUILD.bazel

```diff
@@ -0,0 +1,89 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_library",
+)
+
+rust_library(
+    name = "algebra-snarky",
```

Comment:
I am deeply, deeply unhappy about this. Please tear it out and specify the features where the library is required with those features.

Reply:
done